src/tribol/mesh/CouplingScheme.hpp

```diff
@@ -140,6 +152,8 @@ class CouplingScheme
   CouplingSchemeErrors& getCouplingSchemeErrors() { return m_couplingSchemeErrors; }
   CouplingSchemeInfo&   getCouplingSchemeInfo()   { return m_couplingSchemeInfo; }
 
+  PairReportingData& getPairReportingData() {return m_pairReportingData; }
```

Comment:
Should this be const?

```suggestion
  const PairReportingData& getPairReportingData() const {return m_pairReportingData; }
```

or can the caller modify this?

Reply:
Caller should not modify. Question: the intent of the pair reporting data is that really only the coupling scheme should have control/access to this struct. As is, this getter is not used. Under this design assumption, would you recommend removing it?

Reply:
In that case, we might as well remove this function and at it back later if needed.

Reply:
Fixed.